CHANGELOG.asciidoc

```diff
@@ -38,6 +38,7 @@ Use subheadings with the "=====" level for adding notes for unreleased changes:
 * Capture bucket and object key to Lambda transaction as OTel attributes - `aws.s3.bueckt`, `aws.s3.key` - {pull}3364[#3364]
 * Added `context_propagation_only` configuration option - {pull}3358[#3358]
 * Added attribute[*] for JMX pattern metrics (all metrics can now be generated with `object_name[*:type=*,name=*] attribute[*]`) - {pull}3376[#3376]
+* Added support for OpenTelemetry annotations - `WithSpan` and `CaptureAttribute` - {pull}3406[#3406]
```

Comment:
What is CaptureAttribute?

Reply:
Hi @jackshirazi , 
My apologize, I fixed name of annotation: `CaptureAttribute` -> `SpanAttribute`.